### DISCUSSION TITLE
New Media Type: Link Collection

### DISCUSSION THREAD

**Discussion by inject3d** [stashapp/stash#5096] [Ideas]
It would be great to also store URLs of media from known streaming places within stash. Stash could create groups/categories where a user could store the content. Ideally the most prominent streaming services could be embedded to stash to direct play the video instead of leaving stash. Another subfeature could be to download the video and make it available for offline usage. This feature could be be used for videos or photos.

Reason for this feature is that I have everything at one place and I don't need to create a user at the streaming service to bookmark videos, or to store it as bookmark in the browser.

**Comment by feederbox826** [comment]
https://github.com/stashapp/stash/discussions/5196
